ckanext/publicamundi/plugins.py

```diff
@@ -159,6 +159,10 @@ def before_map(self, mapper):
             m.connect(
                 '/api/publicamundi/dataset/export/{name_or_id}', 
                 action='dataset_export')
+
+            m.connect(
+                '/api/publicamundi/dataset/dcat/{name_or_id}',
```

Comment:
Also, assuming that GeoDCAT transformation is valid only on ISO-derived schemata, maybe we should throw an exception if one tries to apply it on an unrelated schema.